README.md

```diff
@@ -208,6 +208,20 @@ played. The API is of type `SkottieAPI` and provides the following methods:
 |-----------------------|----------------------------------------------------------------------|
 | SkottieAPI.createFrom | Creates a Skottie instance from a source (string, json, file import) |
 
+### Android
+
+#### Proguard
+
+If you're using Proguard, make sure to add the following rule at `proguard-rules.pro`
+
+```
+# for skia, if you haven't add it
+-keep class com.shopify.reactnative.skia.** { *; }
+
+# for skottie
+-keep class com.skiaskottie.** { *; }
+```
+
```

Comment:
Looks good, can you just please add a notice in the Installation section at the bottom - something like:

"For android release builds [see this notice](#markdown-link-to-the-android-section)"

Reply:
Alright. I've moved the section under installation instead, as it makes sense for user to know this after installation.